templates/user/edit-profile.html

```diff
@@ -70,6 +70,25 @@
         .ml-5 {
             margin-left: 5px;
         }
+
+        .hidden-word {
+            padding-top: 5px;
+            font-family: Consolas, Liberation Mono, Monaco, Courier New, monospace;
+            font-size: 12px;
+        }
+
+        .scratch-codes {
```

Comment:
I don't think this CSS applies to anything, since you have `id="scratch-codes"` and this is `.scratch-codes` rather than `#scratch-codes`? Same for `.scratch-codes-text`?

Reply:
(`id` is the correct thing to use here)